### DISCUSSION TITLE
Switching from one UZG01- to another UZG-01 (CC2652P7 chip)

### DISCUSSION THREAD

**Discussion by GD-Dal** [Koenkk/zigbee2mqtt#22837] [General]
Hi 
I have different problems with the UZG-01 [Zigbee GW](https://www.elecrow.com/zigstar-uzg-01-universal-zigbee-gateway.html?sscid=51k8_zgubv&) that I purchased.
But I purchased 2, so I wanted to see if the other one has the same problems.

Can someone tell me a step by step guide on how to switch the 2 boxes without re-pairing all the Zigbee devices?

Thanks

**Comment by francisp2** [comment]
Normally, you should be able to just swap them.
